ydb/docs/en/core/integrations/hibernate.md

```diff
@@ -0,0 +1,237 @@
+# {{ ydb-short-name }} dialect for Hibernate #
+
+## Overview {#overview}
+
+This is a guide to using [Hibernate](https://hibernate.org/) with {{ ydb-short-name }}.
+
+Hibernate is an Object-Relational Mapping (ORM) framework for Java that facilitates the mapping of object-oriented models to SQL. Each database management system (DBMS) partially implements the SQL standard, interpreting certain constructs in its own way. This article describes the specifics of how {{ ydb-short-name }} integrates with Hibernate.
+
+## Installation {#install-dialect}
+
+Add the following dependency to your project:
+
+{% list tabs %}
+
+- Maven
+
+    ```xml
+    <!-- Set an actual versions -->
+    <dependency>
+        <groupId>tech.ydb.jdbc</groupId>
+        <artifactId>ydb-jdbc-driver</artifactId>
+        <version>${ydb.jdbc.version}</version>
+    </dependency>
+
+    <dependency>
+        <groupId>tech.ydb.dialects</groupId>
+        <artifactId>hibernate-ydb-dialect</artifactId>
+        <version>${hibernate.ydb.dialect.version}</version> 
+    </dependency>
+    ```
+
+- Gradle
+
+    ```groovy
+    dependencies {
+        // Set an actual versions
+        implementation "tech.ydb.dialects:hibernate-ydb-dialect:$ydbDialectVersion"
+        implementation "tech.ydb.jdbc:ydb-jdbc-driver:$ydbJdbcVersion"
+    }
+    ```
+
+{% endlist %}
+
+If you use Hibernate version 5, you need `<artifactId>hibernate-ydb-dialect-v5</artifactId>` for Maven or `implementation 'tech.ydb.dialects:hibernate-ydb-dialect-v5:$version` for Gradle.
+
+## Configuration {#configuration-dialect}
```

Comment:
The part about configuring the connection to YDB seems to be missing (endpoint, authentication, etc.).